cmd/server-main.go

```diff
@@ -224,11 +224,6 @@ func serverMain(ctx *cli.Context) {
 	// Handle all server environment vars.
 	serverHandleEnvVars()
 
-	// In distributed setup users need to set ENVs always.
-	if !globalIsEnvCreds && globalIsDistXL {
-		logger.Fatal(uiErrEnvCredentialsMissingServer(nil), "Unable to initialize minio server in distributed mode")
-	}
-
```

Comment:
why remove the check here

Reply:
> why remove the check here

The check is duplicated, it can be found cmd/common-main.go L118

Reply:
ah okay.